api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -256,6 +256,12 @@ message ExternalProcessor {
   // Only one of ``disable_clear_route_cache`` or ``route_cache_action`` can be set.
   RouteCacheAction route_cache_action = 18
       [(udpa.annotations.field_migrate).oneof_promotion = "clear_route_cache_type"];
+
+  // [#not-implemented-hide:]
+  // Specifies the deferred closure timeout for gRPC stream (that connects to external processor). Currently, deferred stream closure is
```

Comment:
I think you should explain in a bit more detail what deferred close is. It needs to be understandable and actionable for a reader who doesn't have deep knowledge of ext_proc filter implementation details, so you need to say something along the lines of "When ext_proc is used in observability mode, streams may be held open to the external processor longer than the lifetime of the regular client to backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this limit". Or something similar along those lines.

Reply:
thank you for suggested comments !

Done.